src/Command/DispatchCommand.php

```diff
@@ -141,12 +141,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     private function updateRepositories(Package $package, array $projectConfig): void
     {
         $repositoryName = $this->getRepositoryName($package);
+        $lastVersion = $this->getLastPackageVersion($package);
+
         $branches = array_keys($projectConfig['branches']);
         $this->io->section('Repository');
 
         $repositoryInfo = $this->githubClient->repo()->show(static::GITHUB_GROUP, $repositoryName);
         $infoToUpdate = [
-            'homepage' => 'https://sonata-project.org/',
+            'homepage' => $lastVersion->getHomepage() ?: 'https://sonata-project.org/',
```

Comment:
IMHO we could use a static value here unless we have different URLs

Reply:
We could use a more specific value: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/composer.json#L11

Reply:
Can you move this to the `projects.yaml` config?

Reply:
Yes I can, but why ?

Shouldn't we try to synchronize the composer.json and the github page ?

Reply:
IMO we should keep the sonata-project.org website here

Reply:
Why sonata-project over the specific bundle site ?
Should we update the one used on packagist then ?

Reply:
@OskarStark Can you explain why ? :)

@sonata-project/contributors WDYT ?

Reply:
I just feel its better to redirect the user to the mainpage of our project website

Reply:
Unless someone starts maintaining the website, redirecting to the mainpage feels like going to an unmaintained project (looks old and outdated). For me, going directly to the docs of the bundle is better